client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Transports.ts

```diff
@@ -248,6 +245,11 @@ export class LongPollingTransport implements ITransport {
 
         this.pollXhr = pollXhr;
         this.pollXhr.open("GET", url, true);
+        if (transferMode === TransferMode.Binary) {
+            pollXhr.responseType = "arraybuffer";
```

Comment:
Most are `this.pollXhr` this one isn't?

Reply:
Moved from a different place where this.pollXhr was not assigned. I will fix for consistency.